services/api/mw2pdf/utils/mediaWikiApi.js

```diff
@@ -0,0 +1,57 @@
+import fetch from 'node-fetch'
+import { toCookieString } from './cookies.js'
+
+export async function makeLoginTokenRequest({ apiUrl }) {
+  const loginTokenRequestParams = new URLSearchParams({
+    'action': 'query',
+    'meta': 'tokens',
+    'format': 'json',
+    'type': 'login',
+  })
+  const loginTokenRequestResult = await fetch(
+    apiUrl,
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded'
+      },
+      body: loginTokenRequestParams.toString(),
+    }
+  )
+  return loginTokenRequestResult
+}
+
+export async function extractLoginToken(loginTokenRequestResult) {
+  const { query: { tokens: { logintoken: loginToken } } } = await loginTokenRequestResult.json()
+  return loginToken
+}
+
+export async function makeLoginRequest({
+  apiUrl,
+  username,
+  password,
+  loginToken,
+  cookies,
+}) {
+  const loginActionRequestBodyData =  {
```

Comment:
This being defined in a separate var was a relic of a debug session and should just be merged into line 44